### PR TITLE
docs: update hambach install to use latest version

### PR DESCRIPTION
**Pull request by dpdanpittman**
## Description

Update validator install instructions to use latest version for hambach.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: docs/validators/get-started/install-regen.md [hunk 1]
```diff
@@ -25,19 +25,19 @@ Check out the genesis version:
...
 ```bash
-git checkout v1.0.0
+git checkout v4.0.0
 ```
 
 *For Redwood Testnet:*
 
 ```bash
-git checkout v1.0.0
+git checkout v4.0.0
```
Comment (ryanchristo) [inline docs/validators/get-started/install-regen.md:34]: `v1.0.0` is the correct version for mainnet and redwood genesis binary.

File: docs/validators/get-started/install-regen.md [hunk 2]
```diff
@@ -71,17 +71,17 @@ You should see the following:
...
 ```bash
-v1.0.0
+v4.0.0
 ```
 
 *For Redwood Testnet:*
 
 ```bash
-v1.0.0
+v4.0.0
```
Comment (ryanchristo) [inline docs/validators/get-started/install-regen.md:80]: Same here. `v1.0.0` is expected for mainnet and redwood genesis. binary.

**Comment by dpdanpittman** [comment]
Good catch!

**Comment by dpdanpittman** [comment]
Fixed and pushed

**Comment by ryanchristo** [comment]
You'll need to update the title of the pull request to use the proper type prefix: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json. For example: https://github.com/regen-network/regen-ledger/pulls.

Also would be good to get in the habit of filling out the description even if the pull request is small. A simple one liner is ok, for example, "Update validator install instructions to use latest version for hamabch".

You should also avoid using the `master` branch on your repository and use the format `dan/update-validator-docs` so that your feature branch is descriptive and you are not limited to one pull request at a time. For example, you can see the format for branch names being used here: https://github.com/regen-network/regen-ledger/branches/all.

**Comment by dpdanpittman** [comment]
Sounds great and thanks for letting me know. I'm just running blind but now
I see!

**Review by ryanchristo — APPROVED** [review]

**Pull request merged**